### PR TITLE
[6.5] Fixed deprecation warning in node test server

### PR DESCRIPTION
**Pull request by GrahamCampbell**
Will allow us to run on node 12 on the master branch when merged upwards.

### CODE REVIEW

**Comment by gmponos** [comment]
> master branch when merged upwards.

Hm.. I am not sure about the process that we will follow.. maybe two different PRs might be needed.

**Comment by GrahamCampbell** [comment]
6.5 is only 5 commits behind master. Should be possible to do a `6.5 -> master` merge. I could probably do it as a PR if you want?

**Comment by GrahamCampbell** [comment]
https://github.com/guzzle/guzzle/pull/2519

File: tests/server.js [hunk 1]
```diff
@@ -178,7 +178,7 @@ var GuzzleServer = function(port, log) {
         that.responses = JSON.parse(request.body);
         for (var i = 0; i < that.responses.length; i++) {
           if (that.responses[i].body) {
-            that.responses[i].body = new Buffer(that.responses[i].body, 'base64');
+            that.responses[i].body = new Buffer.from(that.responses[i].body, 'base64');
```
Comment (guilliamxavier) [inline tests/server.js:181]: you should use either `new Buffer(` or `Buffer.from(`, not `new Buffer.from(` (even if it seems to work...)

Comment (guilliamxavier) [reply tests/server.js:181]: #2638

**Pull request merged**